easybuild/easyconfigs/x/X11/X11-20190313-GCCcore-8.2.0.eb

```diff
@@ -0,0 +1,212 @@
+easyblock = 'Bundle'
+
+name = 'X11'
+version = '20190313'
```

Comment:
@Micket Traditionally, we've used the date of the most recent component as version for the bundle, so this should be `20190311`?